### PR TITLE
Outlier removal loop

### PR DESCRIPTION
**Pull request by sujit22993**
Outlier Removal Added, Bug fixed on no input on first attempt.

Steps to release:
Step 1 : merge this branch into master.
Step 2 : Pull master on production.
Step 3 : Migrate loop_ivr app on production: python manage.py migrate loop_ivr

### CODE REVIEW

File: loop_ivr/utils/config.py [hunk 1]
```diff
@@ -34,10 +34,18 @@
 
 MARKET_INFO_CALL_RESPONSE_URL = 'http://%s%s'%(CURRENT_DOMAIN, '/loopivr/market_info_response/')
 PUSH_MESSAGE_SMS_RESPONSE_URL = 'http://%s%s'%(CURRENT_DOMAIN, '/loopivr/push_message_sms_response/')
-MARKET_INFO_APP = '137265'
+# For production
+# MARKET_INFO_APP = '137265'
```
Comment (Abhishek-lodha) [inline loop_ivr/utils/config.py]: Uncomment production exotel id

File: loop/impact scripts/Aggregator Wise Impact.csv [hunk 2]
```diff
@@ -0,0 +1,53 @@
+Aggregator,Visits Count,Ideal Amount,Total Amount,Total Quantity,Delta %,Per Kg Delta
```
Comment (Abhishek-lodha) [inline loop/impact scripts/Aggregator Wise Impact.csv]: remove csv files

File: loop/impact scripts/Analysis.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
+import pandas as pd
```
Comment (Abhishek-lodha) [inline loop/impact scripts/Analysis.py]: remove impact scripts files. Already integrated in code

File: loop/impact scripts/Aggregator Wise Impact.csv (hunk 2 above)
Comment (sujit22993) [reply loop/impact scripts/Aggregator Wise Impact.csv]: Done : removed

File: loop/impact scripts/Analysis.py (hunk 3 above)
Comment (sujit22993) [reply loop/impact scripts/Analysis.py]: Removed.

File: loop_ivr/utils/config.py (hunk 1 above)
Comment (sujit22993) [reply loop_ivr/utils/config.py]: Done.

**Pull request merged**